app/bundles/CoreBundle/Event/EntityImportEvent.php

```diff
@@ -0,0 +1,120 @@
+<?php
+
+namespace Mautic\CoreBundle\Event;
+
+use Symfony\Contracts\EventDispatcher\Event;
+
+class EntityImportEvent extends Event
+{
+    /**
+     * @var array<int, int>
+     */
+    private array $idMap = [];
+
+    /**
+     * @var array<string, array<string, mixed>>
+     */
+    private array $dependencies = [];
+
+    public const UPDATE = 'update';
+    public const NEW    = 'new';
+    public const ERRORS = 'errors';
+
+    /**
+     * @var array<string, array<string, mixed>>
+     */
+    private array $status = [
+        self::UPDATE => [],
+        self::NEW    => [],
+        self::ERRORS => [],
+    ];
+
+    /** @phpstan-ignore-next-line */
+    public function __construct(private string $entityName, private array $data, private ?int $userId)
+    {
+    }
+
+    public function getEntityName(): string
+    {
+        return $this->entityName;
+    }
+
+    /** @phpstan-ignore-next-line */
+    public function getEntityData(): array
```

Comment:
Please try to resolve each PHPSTAN warning. I guess this is about the array return type. It should be
```suggestion
    /**
    * @return mixed[]
    */
    public function getEntityData(): array
```
Every such ignore ads more tech debt that must be dealt with later. If you must add a PHPSTAN ignore line then also describe why in the comment.

Reply:
fixed